packages/driver-kube-pod/src/driver/client/index.ts

```diff
@@ -177,16 +180,29 @@ const kubeClient = ({ log, namespace, kc, profileId, template, package: packageD
     return await basePortForward({ namespace, forward, log })(podName, targetPort, listenAddress)
   }
 
+  const apiServiceClusterAddress = async (): Promise<[string, number] | undefined> => {
+    const service = await asyncFirst(helpers.listServices({
```

Comment:
Yep, but those env vars are NOT in the compose-tunnel-agent but in the pod running it. so an alternative implementation would be to do run an `exec` on the pod with something like `echo $KUBERNETES_SERVICE_HOST && echo $KUBERNETES_SERVICE_PORT`. I started with that but this seems cleaner

Reply:
I think in Docker-compose is sufficient to use:
env:
- KUBERNETES_SERVICE_PORT
- KUBERNETES_SERVICE_HOST

And it'll forward them if they exist in host

Reply:
if these env vars do not exist, it will print a warning when running the compose file.
they only exist in the case of the kube-pod driver.

Reply:
It’s possible to ignore warning or add them only if Kubernetes pod is used.
Or alternatively, inject them with default values in the compose.
All these solutions are much simpler logically and require little code adjustments. (Especially these that are only in the compose)

Reply:
warnings (like all compose output) are currently piped to stdout, because they might be useful to the user. So we'll have to filter these specific warnings out.

I think it's better to keep driver-specific login in the driver rather than implementing it in the generic packages. IMO the code to retrieve the kubernetes service in the kube-pod driver is not adding local complexity relative to the other stuff that's happening there.